app/components/snippet-list-form.js

```diff
@@ -26,10 +26,29 @@ export default class SnippetListForm extends Component {
     await this.args.model.save();
     this.showSavedTask.perform();
     if (isNew) {
-      this.router.transitionTo('snippets-management.edit', this.args.model);
+      this.router.transitionTo('snippets-management.edit', this.args.model, {
+        queryParams: { showSaved: true },
```

Comment:
still a small nitpick:
I really dislike this `showSaved` name for a variable, because `Saved` could be referring to a snippet list too ("show the saved thing"). I'd rather see `showIsSaved` or `showSavedPill` or anything similar.